pkg/infrastructure/services/kafka.go

```diff
@@ -37,48 +34,46 @@ const (
 	kafkaTopicConfigSegmentKey     string = "segment.bytes"
 	kafkaTopicConfigRetentionValue string = "604800000"
 	kafkaTopicConfigSegmentValue   string = "1073741824"
+
+	quarkusTopicBootstrapEnvVar = "MP_MESSAGING_%s_%s_BOOTSTRAP_SERVERS"
+	quarkusBootstrapEnvVar      = "QUARKUS_KAFKA_BOOTSTRAP_SERVERS"
 )
 
-func setKafkaVariables(externalURI string, container *corev1.Container) {
-	if len(externalURI) > 0 {
-		for _, envKey := range managedKafkaKeys {
-			framework.SetEnvVar(envKey, externalURI, container)
-		}
-	}
-}
+// KafkaTopicMessagingType ...
+type KafkaTopicMessagingType string
```

Comment:
better to define it into deployer.go or ?

Reply:
Totally!